test/verify/check-apps

```diff
@@ -68,7 +68,7 @@ class TestApps(PackageCase):
 </components>
             """.format(body)})
         self.enableRepo()
-        self.machine.execute("pkcon refresh force")
+        self.machine.execute("systemctl stop packagekit && pkcon refresh force")
```

Comment:
We need to do this because PK is buggy, right?

Reply:
Yeah, ideally it would notice changed repos through inotify or so, but apparently it sometimes doesn't. I didn't find another way (through `refresh force` or similar) to get it to "see" the new packages.